config/webpack/webpack.config.js

```diff
@@ -70,6 +71,13 @@ module.exports = function (dirname, config = {}) {
             new CircularDependencyPlugin({
                 exclude: /(node_modules|examples)\/./,
                 failOnError: false // https://github.com/nodejs/readable-stream/issues/280#issuecomment-297076462
+            }),
+            new GlobalizePlugin({
+                production: false,
+                developmentLocale: "en",
+                supportedLocales: ["en", "es", "fr"],
+                messages: "../../src/i18n/[locale].json",
```

Comment:
Yes, we can use loadMessages repetively like:
Globalize.loadMessages(JSON.parse(fs.readFileSync(filename, 'utf8')));
It will override the message if it has the same definition and add new ones as well.
The only problem as you describe is the filename path. How do we determine the location of the filepath?
Relative path: "../../src/...  Do we have a variable that tells us where to start the path for the new Globalize JSON file for each package ?? We need to be able to find the exact location of the file before loading it to the application. The value for "${process.cwd()}" tell us from where we initiated THEIA, but this value could be different  depending where yo are to start the application.

- I agree multi-level is good for the variable definition.